.github/workflows/docs-to-gh-wiki.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     - name: Push Wiki Changes
       uses: Andrew-Chen-Wang/github-wiki-action@v3
       env:
-        WIKI_DIR: docs/
+        WIKI_DIR: docs/wiki
```

Comment:
This needs to end in a `/` as it uses rsync.

Reply:
nice catch